src/applications/appeals/10182/config/form.js

```diff
@@ -21,8 +21,8 @@ import {
   wantsToUploadEvidence,
   needsHearingType,
   appStateSelector,
-  getIssueName,
```

Comment:
Does `getIssueName` get used anywhere else or can it be officially deleted from `../utils/helpers` if it still exists?